scarlet/source.py

```diff
@@ -59,17 +61,68 @@ def get_psf_sed(sky_coord, observation, frame):
     sed = get_pixel_sed(sky_coord, observation)
 
     # approx. correct PSF width variations from SED by normalizing heights
+    if observation._diff_kernels is not None:
+        sed /= observation._diff_kernels.image.sum(axis=(-2,-1))
+
+        return sed
+
     if observation.frame.psf is not None:
         # Account for the PSF in the intensity
-        sed /= observation.frame.psf.image.max(axis=(1, 2))
+        sed /= observation.frame.psf.image.sum(axis=(-2, -1))
 
     if frame.psf is not None:
-        sed = sed * frame.psf.image[0].max()
+        sed = sed * frame.psf.image[0].sum()
 
     return sed
 
+def build_sed_coadd(seds, bg_rmses, observations):
```

Comment:
this functions call all go to `initialization.py` (note the spelling...) as well